src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+@import "../utils/variables";
+
+.stars {
+  display: flex;
+  // padding: 16px 16px 0;
```

Comment:
Remove the commented-out padding property if it's not needed in the .stars class.